### PR TITLE
[WIP] Edit profile with Devise

### PR DESCRIPTION
**Pull request by iamPhong**
- [x] edit profile with devise

- [x] custom view edit

- [x] review avatar before upload

### CODE REVIEW

**Comment by truong-dat-mulodo** [comment]
@iamPhong Please update title of pull request with [WIP] _(work in progress)_
Write description for this pull ( In this pull request, what are you doing?  )

**Comment by iamPhong** [comment]
Ready

File: app/assets/javascripts/custom.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
+  $('body').on('click', function() {
+    $('.sidebar').removeClass('opened');
+  });
+
+  $('body').on('click', '.is-hover #btn-list-lease', function(e) {console.log('SS');
```
Comment (huongnguyenmta) [inline app/assets/javascripts/custom.js]: bỏ console.log đi em

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,58 +1,135 @@
...
+        <div class="form-group">
+          <%= f.label :phone_number, class:"control-label col-sm-2" %>
+          <div class="col-sm-3">
+            <%= f.text_field :phone_number, autofocus: true, autocomplete: "phone_number", class:"form-control" %>
+          </div>
+        </div>
+        <div class="form-group">
+          <%= f.label :facebook, class:"control-label col-sm-2" %>
+          <div class="col-sm-10">
+            <div class="col-sm-2 text-right" style="padding: 6px 0px;">http://facebook.com/</div>
```
Comment (huongnguyenmta) [inline app/views/devise/registrations/edit.html.erb]: khai báo style trong file css đi em,
ko khai báo ở file view như này nhé

File: app/views/shared/_sidebar.html.erb [hunk 3]
```diff
@@ -0,0 +1,139 @@
+<div class="col-sm-3 col-md-3 col-lg-3 sidebar">
+  <div class="i-want is-hover">
+    <button class="btn btn-warning" id="btn-list-lease" type="button">
+      <i class="fa fa-list-ul"></i>
+      <%= t ".iwant" %>
+    </button>
+    <ul class="nav list-lease-box cfi-iwant">
+      <li class="list-type-button types-1" data-id="1">
+        <%= link_to "#",class:"cfi-open-child cfi-type-a" do %>
+          <%= image_tag("sc1.png", width:"20px") %>
```
Comment (huongnguyenmta) [inline app/views/shared/_sidebar.html.erb]: tương tự nhé em

**Comment by iamPhong** [comment]
Ready

**Comment by iamPhong** [comment]
Ready

**Pull request merged**